OpenGL/Math/Matrix4.cs

```diff
@@ -108,6 +108,82 @@ public static Matrix4 Identity
             }
         }
 
+        public float this[int column, int row]
+        {
+            get
+            {
+                if (row == 0)
+                {
+                    if (column == 0) return row1.X;
+                    if (column == 1) return row1.Y;
+                    if (column == 2) return row1.Z;
+                    if (column == 3) return row1.W;
+                }
+                if (row == 1)
+                {
+                    if (column == 0) return row2.X;
+                    if (column == 1) return row2.Y;
+                    if (column == 2) return row2.Z;
+                    if (column == 3) return row2.W;
+                }
+                if (row == 2)
+                {
+                    if (column == 0) return row3.X;
+                    if (column == 1) return row3.Y;
+                    if (column == 2) return row3.Z;
+                    if (column == 3) return row3.W;
+                }
+                if (row == 3)
+                {
+                    if (column == 0) return row4.X;
+                    if (column == 1) return row4.Y;
+                    if (column == 2) return row4.Z;
+                    if (column == 3) return row4.W;
+                }
+                if (column > 3 || column < 0 || row > 3 || row < 0)
```

Comment:
Normally checking inputs for being out of range will happen at the start of the method/etc.